src/controllers/token.rs

```diff
@@ -138,9 +140,26 @@ pub async fn new(app: AppState, req: BytesRequest) -> AppResult<Json<Value>> {
             endpoint_scopes,
             new.api_token.expired_at,
         )?;
-        let api_token = EncodableApiTokenWithToken::from(api_token);
 
-        Ok(Json(json!({ "api_token": api_token })))
+        if let Some(recipient) = recipient {
+            // At this point the token has been created so failing to send the
+            // email should not cause an error response to be returned to the
+            // caller.
+            let email_ret = app.emails.send(
+                &recipient,
+                NewTokenEmail {
+                    user_name: &user.gh_login,
+                    domain: &app.emails.domain,
+                },
+            );
+            if let Err(e) = email_ret {
+                error!("Failed to send token creation email: {e}")
+            }
```

Comment:
I figured it'd be better to not cause token creation failures if the email service is throwing errors, so I put it _after_ token creation and the API still returns the OK response it does today. I can change this to a hard error if you prefer though 🙏